ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -427,7 +427,32 @@ aws|aws-arm64|aws-usgov)
     ;;
 esac
 
-sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
+function setup-cvo() {
+    local payload_version
+    payload_version="$(oc adm release info "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}" -o "jsonpath={.metadata.version}")"
+    echo "Release payload version: ${payload_version}"
+    local channel
+    channel="$(grep -E --only-matching '(stable|eus|fast|candidate)-4.[0-9]+' "${dir}/manifests/cvo-overrides.yaml")"
+    echo "Original channel from CVO manifest: ${channel}"
+    local candidate_channel
+    # WTF THIS SHOULD WORK candidate_channel="${channel//+(stable|eus|fast)/candidate}"
+    candidate_channel="$(echo "${channel}" | sed -E 's/(stable|eus|fast)/candidate/')"
+    echo "Matching candidate channel: ${candidate_channel}"
+
+    if curl --silent "https://api.integration.openshift.com/api/upgrades_info/v1/graph?channel=$candidate_channel" | grep --quiet '"version":"'"$payload_version"'"'; then
+        echo "Version ${payload_version} is available in ${candidate_channel}: cluster can query OSUS"
```

Comment:
nit: I'd rather spell out "the OpenShift Update Service", to help out folks who haven't internalized "OSUS" (probably most readers ;) )